central/requirements.yaml

```diff
@@ -5,7 +5,7 @@ dependencies:
 #  repository: "file://../centralhub"
 #  condition: centralhub.enabled
 - name: centralledger
-  version: 3.8.5
```

Comment:
Shouldn't this be 3.8.6?

https://github.com/mojaloop/helm/pull/111/files <-- I suspect you didn't pull master into your feature branch.

Reply:
Hmm, I did pull in master and created this one off of it. I was surprised to see 3.8.5 there as well.. here is the change in that file in PR#111 - https://github.com/mojaloop/helm/pull/111/files#diff-da85c34fd389e4b583301a1bb55645de (the version wasn't updated to 3.8.6 there)

Reply:
Please let me know if I can go ahead with the merge..

Reply:
seems like there was some inconsistency with the version numbers previously. Looks ok to me